app/user_agent_test.go

```diff
@@ -53,9 +54,8 @@ func TestGetPlatformName(t *testing.T) {
 		t.Run(fmt.Sprintf("GetPlatformName_%v", i), func(t *testing.T) {
 			ua := uasurfer.Parse(userAgent.UserAgent)
 
-			if actual := getPlatformName(ua); actual != expected[i] {
-				t.Fatalf("%v Got %v, expected %v", userAgent.Name, actual, expected[i])
-			}
+			actual := getPlatformName(ua)
+			require.Equalf(t, actual, expected[i], "%v Got %v, expected %v", userAgent.Name, actual, expected[i])
```

Comment:
A few things to note here:
1. The "Got X, expected Y" message for these tests is actually unnecessary because `Equal` prints the actual and expected when a test fails anyway. Can you remove the message from these and change them from `Equalf` to `Equal`?
2. `require` is only supposed to be used for setup code. Instead, we should be using `assert`. Would you mind changing that as well?
3. These methods want the expected value first, so the actual and expected arguments should be swapped.

Would you be able to update these tests? For example, this line would be 
```suggestion
			assert.Equal(t, expected[i], actual)
```

Reply:
done